docs/DeveloperGuide.adoc

```diff
@@ -795,38 +795,209 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
+
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new client | track the client
+
+|`* * *` |user |add multiple client | track the number of clients I have
+
+|`* * *` |user |remove my client | remove entries that I no longer need
+
+|`* * *` |user |add the appointment date I have with my client |keep track of the next meet up
+
+|`* * *` |user |sort the client based on different category |directly get the type of data I want
+
+|`* * *` |user |add my client’s personal detail  |know more about them
+
+|`* * *` |user |update my client's personal detail |keep their information up to date
+
+|`* * *` |user |search for a single client  |narrow down to the client I am interested in in that time
+
+|`* * *` |user |clear the address book |start on a clean slate
+
+|`* * *` |user |exit my application |exit the application once I am done with it
+
+|`* * *` |user |add remarks of my client |remember my personal comment of the client
+
+|`* * *` |user |add the insurance plans that my clients are committed |track what plans he/she has
```

Comment:
I don't see any user story that cover the addition of insurance plan? I don't know how it will refer to the selected plan.